.github/workflows/jira-issue-create.yml

```diff
@@ -0,0 +1,34 @@
+name: Jira Issue Creator
+
```

Comment:
since this is an open-source repo, maybe a description is good for those prying eyes looking into our workflows

Reply:
👍